tests/utils/test_utils.py

```diff
@@ -117,6 +117,26 @@ def test_readfile_binary(self):
         assert content.splitlines()[0] == b"Hello World!"
 
 
+class TestResourceFilename:
+
+    """Test resource_filename."""
+
+    @pytest.fixture(autouse=True, params=[True, False])
+    def freezer(self, request, monkeypatch):
```

Comment:
Can you instead move the existing `freezer` to the module level, remove `autouse=True`, and then add `@pytest.mark.usefixtures('freezer')` to `TestResourceFilename` and `TestReadFile`?